From 37700fd0ceefcdc6282fa531dd143f54ccc73217 Mon Sep 17 00:00:00 2001
From: Ken Tseng <ken.tseng@realtek.com>
Date: Thu, 5 Feb 2026 14:20:37 +0800
Subject: [PATCH] arm64: kvm: Makefile.nvhe: Use O2 for 128bit shift in clock.c

arch/arm64/kvm/hyp/nvhe/clock.c:39:undefined reference to __kvm_nvhe___lshrti3
---
 arch/arm64/kvm/hyp/nvhe/Makefile.nvhe | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/kvm/hyp/nvhe/Makefile.nvhe b/arch/arm64/kvm/hyp/nvhe/Makefile.nvhe
index 977357a9c4c5f..35b8dd527aba2 100644
--- a/arch/arm64/kvm/hyp/nvhe/Makefile.nvhe
+++ b/arch/arm64/kvm/hyp/nvhe/Makefile.nvhe
@@ -13,6 +13,7 @@ ccflags-y := -D__KVM_NVHE_HYPERVISOR__ -D__DISABLE_EXPORTS -D__DISABLE_TRACE_MMI
 ccflags-y += -fno-stack-protector	\
 	     -DDISABLE_BRANCH_PROFILING	\
 	     $(DISABLE_STACKLEAK_PLUGIN)
+ccflags-y += -O2
 
 HYPREL := arch/arm64/tools/gen-hyprel
 
-- 
2.34.1

